utils/transaction-provider/cosmos/cosmos-tx-provider.ts

```diff
@@ -0,0 +1,268 @@
+import { create } from "@bufbuild/protobuf";
+import { TW, type WalletCore } from "@trustwallet/wallet-core";
+import type { CoinType } from "@trustwallet/wallet-core/dist/src/wallet-core";
+import Long from "long";
+import {
+  CosmosSpecificSchema,
+  MAYAChainSpecificSchema,
```

Comment:
`MAYAChainSpecificSchema`, `MAYAChainSpecific` is not used in here, could you please clean it up?